ffi/README.md

```diff
@@ -4,13 +4,21 @@ FFI wrapper for iotics-identity-go
 
 ## Build
 
-`make compile`
+With `make` installed you can compile for the current platform with
+`make compile`. The script will detect the OS and ARCH and build for your system.
 
-The target should determine the platform and architecture of the current OS and build the binary library accordingly.
+If you want to override the target os (the `GOOS` variable), define `$OS_FLAG` to the desired value.
+For example if you want to compile for an iOS target run
+`env OS_FLAG=ios make compile`
 
-Alternatively you can build your version using:
 
-`env GOOS=<os> GOARCH=<arch> go build -buildmode=c-shared -o lib/lib-iotics-id-sdk.so ./ffi_wrapper.go`
+On Windows, if you don't have `make` you can run the powershell script `build.ps1` like:
+
+`powershell -ExecutionPolicy Bypass -File build.ps1`
+
+Alternatively you can build your version manually using:
+
+`env GOOS=<$OS_FLAG> GOARCH=<$OS_ARCH> go build -buildmode=c-shared -o lib/lib-iotics-id-sdk.so ./ffi_wrapper.go`
```

Comment:
should `lib/lib-iotics-id-sdk.so` be `.dll`?

Reply:
Should we discourage  this manual mode?
(The PS script doesn't really do anything - I imagine in CIs it's more like you can exec a `.bat` file, not PS. Or am I wrong and these days PS is expected to exist everywhere?)

Reply:
I am on windows 11 and by default I get powershell which has loads of installed and useful commands. PS over Bat.
regarding building manually vs using a script, I don't see why this line poses any problem 

addressed the so/dll in comment